### ISSUE TITLE
ResNet18 d8

### ISSUE THREAD

**Issue by ph-code-repo** [dmklee/equivision#1]
Hi there,
Would it be possible for you to train a d8 ResNet18 network for me.
I'd really like to be able to have the same number of rotations as the C8 network, but within the dihedral group.
I doubt that I have the resources to be able to train it myself.

Kind Regards!

**Comment by dmklee** [comment]
Yes, I'd be happy to train that for you.  I can create a `d8resnet` using a `base_width=20` which comes out to around the same number of trainable parameters as the other models.  Here are some stats for it:

Num Params: 11.9M
Eval forward pass (batchsize=1): 25ms | 3.4 Gb 
Train forward pass (batchsize=32): 230ms | 8.4 Gb

I got these numbers with an RTX2080Ti.  Does this sound like it would work for your situation? It will take a few days to train so I want to confirm it's right for your application.

**Comment by ph-code-repo** [comment]
That is great news, thank you! Just to check, is the `base_width` parameter the number of channels/kernels used in the first layer of ResNet, so that the number in the final layer of ResNet 18 would be 8 times this amount? In which case, I think this should be fine. 
Thank you for responding so quickly!

**Comment by dmklee** [comment]
It's training now, sorry for the delay.  I'll upload the checkpoint by the end of the week.
